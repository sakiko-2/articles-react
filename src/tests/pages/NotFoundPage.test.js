import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../pages/NotFoundPage';

it('renders NotFoundPage correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});

it('renders Page Not Found text', () => {
  const wrapper = shallow(<NotFoundPage />);
  const notFound = <h1 className="title">Page Not Found</h1>;
  expect(wrapper.contains(notFound)).toEqual(true);
});
