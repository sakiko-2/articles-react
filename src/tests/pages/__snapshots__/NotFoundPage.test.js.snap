// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NotFoundPage correctly 1`] = `ShallowWrapper {}`;
